subprojects/core/src/test/groovy/org/gradle/api/internal/tasks/DefaultPropertyWalkerTest.groovy

```diff
@@ -51,6 +51,7 @@ class DefaultPropertyWalkerTest extends AbstractProjectBuilderSpec {
         1 * visitor.visitInputFileProperty({ it.propertyName == 'inputFile' })
         1 * visitor.visitInputFileProperty({ it.propertyName == 'inputFiles' })
         1 * visitor.visitInputProperty({ it.propertyName == 'bean.class' && it.value == NestedBean.name })
+        1 * visitor.visitInputProperty({ it.propertyName == 'bean.$$implementation$$' })
```

Comment:
This is personal taste, but the `$$` suffix gives me a bad tick. :) Can we just do one or two `$`s as a prefix?

Reply:
No more `$$` suffix it is.